torchvision/transforms/v2/functional/_meta.py

```diff
@@ -352,14 +352,122 @@ def _clamp_bounding_boxes(
     return out_boxes.to(in_dtype)
 
 
+def _order_bounding_boxes_points(
+    bounding_boxes: torch.Tensor, indices: torch.Tensor | None = None
```

Comment:
I think the `|` operator for annotations will fail on 3.9. Use `Union`, or `from __future__ import annotations`